api/config/jwt.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | JWT Authentication Secret
+    |--------------------------------------------------------------------------
+    |
+    | The paths to the RSA keypair used to sign tokens.
+    |
+    */
+
+    'secret' => [
+        'public' => storage_path('app/keys/public.pem'),
+        'private' => storage_path('app/keys/private.pem')
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | JWT time to live
+    |--------------------------------------------------------------------------
+    |
+    | Specify the length of time (in minutes) that the token will be valid for.
+    | Defaults to 1 hour
+    |
+    */
+
+    'ttl' => 60,
+
+    /*
+    |--------------------------------------------------------------------------
+    | Refresh time to live
+    |--------------------------------------------------------------------------
+    |
+    | Specify the length of time (in minutes) that the token can be refreshed
+    | within. I.E. The user can refresh their token within a 2 week window of
+    | the original token being created until they must re-authenticate.
+    | Defaults to 2 weeks
+    |
+    */
+
+    'refresh_ttl' => 20160,
```

Comment:
Not really, though the underlying JWT package looks for this config attribute, so if we remove it, we're going to have to override those package methods to remove that part of the code.